Model/CustomItemExportSchedulerModel.php

```diff
@@ -0,0 +1,271 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MauticPlugin\CustomObjectsBundle\Model;
+
+use DateTimeImmutable;
+use DateTimeZone;
+use Mautic\CoreBundle\Helper\ExportHelper;
+use Mautic\CoreBundle\Model\AbstractCommonModel;
+use Mautic\EmailBundle\Helper\MailHelper;
+use MauticPlugin\CustomObjectsBundle\Entity\CustomField;
+use MauticPlugin\CustomObjectsBundle\Entity\CustomItem;
+use MauticPlugin\CustomObjectsBundle\Entity\CustomItemExportScheduler;
+use MauticPlugin\CustomObjectsBundle\Entity\CustomObject;
+use MauticPlugin\CustomObjectsBundle\Provider\CustomItemRouteProvider;
+use MauticPlugin\CustomObjectsBundle\Repository\CustomItemExportSchedulerRepository;
+use MauticPlugin\CustomObjectsBundle\Repository\CustomItemRepository;
+use MauticPlugin\CustomObjectsBundle\Repository\CustomItemXrefContactRepository;
+use Symfony\Component\HttpFoundation\BinaryFileResponse;
+use Symfony\Component\HttpFoundation\Response;
+
+class CustomItemExportSchedulerModel extends AbstractCommonModel
+{
+    private const EXPORT_FILE_NAME_DATE_FORMAT = 'Y_m_d_H_i_s';
+    private const CUSTOM_ITEM_LIMIT            = 200;
+    private const CONTACT_LIMIT                = 1000000;
```

Comment:
Also can you please confirm if 1M comma separated values get  stored in single cell. IMO it needs to be low.